frame/nfts/src/lib.rs

```diff
@@ -306,6 +306,12 @@ pub mod pallet {
 	pub(super) type CollectionMaxSupply<T: Config<I>, I: 'static = ()> =
 		StorageMap<_, Blake2_128Concat, T::CollectionId, u32, OptionQuery>;
 
+	#[pallet::storage]
+	/// Stores the `CollectionId` that is going to be used for the next collection.
+	/// This gets incremented by 1 whenever a new collection is created.
+	pub(super) type NextCollectionId<T: Config<I>, I: 'static = ()> =
+		StorageValue<_, T::CollectionId, OptionQuery>;
```

Comment:
Is this bit in the right PR - is it meant to be here?

Reply:
It was already there, I didn't add it in the current PR